pkg/collector/corechecks/embed/process_agent.go

```diff
@@ -127,9 +127,10 @@ func (c *ProcessAgentCheck) run() error {
 
 // Configure the ProcessAgentCheck
 func (c *ProcessAgentCheck) Configure(data integration.Data, initConfig integration.Data, source string) error {
-	// handle the case when the agent is disabled via the old `datadog.conf` file
+	// only log whether process check is enabled or not but don't return early, because we still need to initialize "binPath", "source" and
+	// start up process-agent. Ultimately it's up to process-agent to decide whether to run or not based on the config
 	if enabled := config.Datadog.GetBool("process_config.enabled"); !enabled {
-		log.Info("Process Agent disabled through main configuration file")
+		log.Info("process-check is disabled through main configuration file")
```

Comment:
This is still misleading as there is a `process` integration that customers and support might typically refer to as the `process check`.